src/main/java/net/fabricmc/loader/game/MinecraftGameProvider.java

```diff
@@ -158,6 +159,31 @@ public void acceptArguments(String... argStrs) {
 		FabricLauncherBase.processArgumentMap(arguments, envType);
 	}
 
+	@Override
+	public String[] getLaunchArguments(boolean sanitize) {
+		if (arguments != null) {
+			List<String> list = new ArrayList<>(Arrays.asList(arguments.toArray()));
+			if (sanitize) {
```

Comment:
Empty line before opening this `if` statement. Ideally we want to keep this close to fabric api codestyle but we have a lot to do in loader before we pull over checkstyle.

Reply:
Done!